app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureController.scala

```diff
@@ -52,10 +52,11 @@ class CaptureBox5FigureController @Inject()(mcc: MessagesControllerComponents,
               Future.successful(
                 BadRequest(view(formWithErrors, routes.CaptureBox5FigureController.submit(journeyId)))
               ),
-            box5Figure =>
+            box5Figure => {
```

Comment:
Do we need braces added here? None of the other controllers are formatted like this